cpp/src/io/parquet/page_enc.cu

```diff
@@ -396,7 +396,9 @@ __device__ size_t delta_data_len(Type physical_type, cudf::type_id type_id, uint
   // need max dtype_len + 1 bytes for min_delta
   // one byte per mini block for the bitwidth
   // and block_size * dtype_len bytes for the actual encoded data
-  auto const block_size = dtype_len + 1 + delta::num_mini_blocks + vals_per_block * dtype_len;
+  auto const max_bits        = dtype_len * 8 + 1;
+  auto const bytes_per_block = max_bits * vals_per_block / 8;
```

Comment:
should this be rounded up?
```suggestion
  auto const bytes_per_block = cudf::util::div_rounding_up_unsafe(max_bits * vals_per_block, 8);
```

Reply:
Not really a concern because vals_per_block will always be a multiple of 8, but will change it.

Reply:
I really thought I got something there :D